hicommonwealth/commonwealth
# 6960 breadcrumb pwa bug

## Link to Issue
Closes: #6960 

## Description of Changes
- Added a condition to look for sidebar

## "How We Fixed It"

- Using the `useSidebarStore` hook I set the css based on the sidebar visibility. 

## Test Plan
- Go to any thread
- Hover over the first breadcrumb 
- Open and close the sidebar
- Ensure padding is consistent
